src/common/src/system_param/common.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Mutex;
+
+use super::reader::SystemParamsReader;
+use crate::util::tracing::layer::toggle_otel_layer;
+
+/// Node-independent handler for system parameter changes.
```

Comment:
What does "Node-independent handler" mean? What's node-dependent?

Reply:
Some system param subscribers are only responsible for some specific business of that node.

- Compute:
https://github.com/risingwavelabs/risingwave/blob/99fa11570ccc58dd02899927d291b59aa2363cf5/src/compute/src/server.rs#L294-L298

- Meta:
https://github.com/risingwavelabs/risingwave/blob/99fa11570ccc58dd02899927d291b59aa2363cf5/src/meta/src/barrier/mod.rs#L737-L740